main.js

```diff
@@ -107,17 +127,7 @@ function alienChoiceDifficult() {
   game.determineWinner("alien");
 };
 
-function difficultGamePage(){
-  game.type = "difficult";
-  hide([mainPage, winnerWrapDifficult, resultTextDifficult]);
-  show([difficultPageView, difficultGameSection, chooseFighterTextDifficult, changeGameBtn]);
-};
-
-function displayWinnerDifficult(){
-  show([winnerWrapDifficult, resultTextDifficult]);
-  hide([difficultGameSection, chooseFighterTextDifficult, changeGameBtn]);
-};
-
+// Show/Hide
 function hide(elements) {
```

Comment:
Turning the parameter into an array was an excellent choice! Way to dry that up!